server/routes/aws/tasks.py

```
@@ -0,0 +1,121 @@
+    Extracts affected resources to build an actionable checklist.
+    """
+    title = finding.get("Title", "Unknown Finding")
+    desc = finding.get("Description", "")
+    severity = finding.get("Severity", {}).get("Label", "UNKNOWN")
+    
+    resources = finding.get("Resources", [])
+    
+    resource_names = []
+    service_types = []
+    for res in resources:
+        if res.get("Id"): resource_names.append(res["Id"])
```

Ruff (0.15.10)

[error] Multiple statements on one line (colon)

(E701)